init.c

```diff
@@ -276,10 +278,12 @@ static void print_help()
 	  "                                additional -w adds more URLs\n"
 	  "-x, --cross-seed              : ensure info hash is unique for easier cross-seeding\n"
 #else
-	  "-a <url>[,<url>]* : specify the full announce URLs\n"
+	  "-a <url>[,<url>]* : sdpecify the full announce URLs\n"
```

Comment:
Please fix this.

Reply:
Sorry, don't know where that came from. Fixed.